I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 654, in script
    if claudia_good:
  File "game/script.rpy", line 658, in <module>
    elif seeing_gaster:
NameError: name 'seeing_gaster' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 654, in script
    if claudia_good:
  File "/Users/lillydivack/Desktop/renpy-8.2.1-sdk/renpy/ast.py", line 1842, in execute
    if renpy.python.py_eval(condition):
  File "/Users/lillydivack/Desktop/renpy-8.2.1-sdk/renpy/python.py", line 1209, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "/Users/lillydivack/Desktop/renpy-8.2.1-sdk/renpy/python.py", line 1202, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 658, in <module>
    elif seeing_gaster:
NameError: name 'seeing_gaster' is not defined

macOS-14.3.1-arm64-arm-64bit arm64
Ren'Py 8.2.1.24030407
tribehacks 1.0
Sun Apr 21 08:49:02 2024
